container/Makefile

```diff
@@ -1,17 +1,25 @@
+#Configure your docker image name with registry url
 DOCKER_IMG=aruba-iotops-example-ble:$(shell head ../VERSION)
 
-ifeq ($(shell uname),Darwin)
-DOCKER_FLAGS +=--platform=linux/amd64
-endif
+.PHONY: build
+build:
+	go build -o app ./application
 
 .PHONY: test
 test:
 	go test -v ./application
 
-.PHONY: build
-build:
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags '-w -s'  -o ./app ./application
+.PHONY: docker-local
+docker-local:
+	docker buildx build --load -t $(DOCKER_IMG) .
+
+#Support both data collector and AP as connector
+IMAGE_PLATFORM +=--platform=linux/amd64,linux/arm64,linux/arm/v7
+.PHONY: publish-docker
+publish-docker:
+	docker buildx build ${IMAGE_PLATFORM} --push -t $(DOCKER_IMG) .
 
-.PHONY: docker
-docker: build
-	docker build ${DOCKER_FLAGS}  -f ./Dockerfile -t $(DOCKER_IMG) .
+#Build a tar to upload in Aruba Developer Portal
+.PHONY: docker-tar
+docker-tar:
+	docker buildx build ${IMAGE_PLATFORM} -o - . > images.tar
```

Comment:
this command needs to be changed to specify type as oci
ex.: docker buildx build ${IMAGE_PLATFORM} -o type=oci,dest=./images.tar .